API.md

```diff
@@ -510,6 +511,12 @@ GetContainer returns information about a single container.  If a container
 with the given id doesn't exist, a [ContainerNotFound](#ContainerNotFound)
 error will be returned.  See also [ListContainers](ListContainers) and
 [InspectContainer](#InspectContainer).
+### <a name="GetContainerByID"></a>func GetContainerByID
+<div style="background-color: #E8E8E8; padding: 15px; margin: 10px; border-radius: 10px;">
+
+method GetContainerByID(id: [string](https://godoc.org/builtin#string)) [string](https://godoc.org/builtin#string)</div>
+GetContainerByID allows you to get a container id from its partial or full name
+or partial ID
```

Comment:
I'd love to have an example here explaining the difference to `GetContainer()`

Reply:
In fact, it's very confusing to call it `GetContainerByID` but return a return a `string`. `GetContainer...` makes me intuitively expect to get a container. @marcov, can't we use the existing API?

Reply:
Maybe rename to `ResolveContainerID`?

Reply:
`ResolveContainerID` sounds good to me. 
@vrothberg, it's not possible to use the `GetContainersByContext` because that would fail the container config is corrupt.
I don't stricly need to add a new public API, I assumed that's the only way to call libpod functions from `containers_remote.go:RemoveContainers(...)`. I am open to do differently if possible.

Reply:
@marcov is it possible to let `EvictContainer` accept partial IDs?

Reply:
@vrothberg, yep that's a possibility. I'd move `the call to LookupContainerID` in it so that the above API would not be needed anymore.

Do you confirm this is the preferred solution?

Reply:
Thanks! Yes, I prefer to use the existing API over adding new ones.